### ISSUE TITLE
Naming: Grip icon

### ISSUE THREAD

**Issue by zoharma** [lucide-icons/lucide#1467]
This is about the naming convention for your `grip` (drag) icon. Commonly, a grid of 9 dots is not used for dragging, but rather used as a way to switch and navigate between multiple apps.

Icon location: https://lucide.dev/icons/grip

Can be seen here:
https://fonts.google.com/icons?selected=Material%20Symbols%20Outlined%3Aapps%3AFILL%400%3Bwght%40400%3BGRAD%400%3Bopsz%4048
This can also be found in Atlassian, Apple SF Symbols, and many more

**Comment by jguddas** [comment]
Yeah, not the best name, +1 for transition all icons to descriptive names ie. like when we renamed `edit` to `pen-square`. 

Maybe `grid-9-dots` or something IDK.
